x-pack/plugin/esql/qa/testFixtures/src/main/resources/categorize.csv-spec

```diff
@@ -190,9 +202,9 @@ FROM sample_data
 ;
 
 COUNT():long | category:keyword
-           1 | .*?\.\*\?Connected\.\+\?to\.\*\?.*?
-           1 | .*?\.\*\?Connection\.\+\?error\.\*\?.*?
-           1 | .*?\.\*\?Disconnected\.\*\?.*?
```

Comment:
The categorization analyzer is more aggressive w.r.t. specials chars, hence these changes.